include/snowflake/client.h

```diff
@@ -403,6 +419,11 @@ typedef struct SF_TIMESTAMP {
     SF_DB_TYPE ts_type;
 } SF_TIMESTAMP;
 
+/**
+ * Initializes an SF_QUERY_RESPONSE_CAPTURE struct.
+ */
+SF_QUERY_RESULT_CAPTURE snowflake_query_result_capture_init();
```

Comment:
Also this is missing STDCALL from the function definition (don't forget to add that in the .c file as well). And can we ad a term function for this?